### DISCUSSION TITLE
Is it possible to have space around segments?

### DISCUSSION THREAD

**Discussion by kakulukia** [starship/starship#1432] [General]
Im struggling to find a way to have spaces around for example the username.

Is that possible at all?
Im speaking about something like this:

<img width="82" alt="Bildschirmfoto 2020-07-05 um 11 46 54" src="https://user-images.githubusercontent.com/96352/86530347-59519480-beb8-11ea-97fe-ecee39d98d95.png">

**Comment by andytom** [comment]
At the moment this is possible for some modules but not all and when possible it is not always consistent, we are very close to the release the format string feature which will make this sort of customisation a lot easier, if you can track the progress in #1057.

**Reply by kakulukia** [reply]
Thx! Looking forward to that release!
